demo_nodes_cpp/src/services/add_two_ints_client.cpp

```diff
@@ -65,9 +65,15 @@ int main(int argc, char ** argv)
   }
 
   auto topic = std::string("add_two_ints");
+
   if (rcutils_cli_option_exist(argv, argv + argc, "-s")) {
```

Comment:
we might as well remove this check now that we do not assume that `rcutils_cli_get_option` will return a valid string.